src/test/scala/is/hail/methods/FilterSuite.scala

```diff
@@ -40,7 +40,7 @@ class FilterSuite extends SparkSuite {
 
     assert(sQcVds.filterSamplesExpr("sa.qc.nCalled == 337").nSamples == 17)
 
-    assert(sQcVds.filterSamplesExpr("sa.qc.dpMean > 60").nSamples == 7)
+    assert(sQcVds.filterSamplesExpr("sa.qc.dpMean > 60").nSamples == 6)
```

Comment:
Double check 6 should be the new correct value and this isn't an artifact of having the wrong type in the GenotypeExtractor.